command/operator_diagnose.go

```diff
@@ -14,6 +14,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/openbao/openbao/helper/constants"
```

Comment:
I'd run `make fmt` if you don't mind!

Reply:
Oh interesting, I guess you did run `make fmt` but that didn't reorganize the inputs. Huh, TIL!